std/process.d

```diff
@@ -1266,6 +1266,62 @@ final class Pid
         return _processID;
     }
 
+    version (Windows)
+    {
+        /**
+        A simple factory that returns a $(LREF Pid) handle when provided with a
+        global `pid`. This handle can be used to kill processes that haven't been
+        spawned by this process.
+        Note: this does not check whether the provided `pid` is valid and
+        corresponds to a running process within the operating system.
+
+        Params:
+            pid = a process id
+            desiredAccess = (Windows specific, optional) the desired access to the
+                            process object
+
+        Returns: an instance to a `Pid` that corresponds to the pid within the
+        operating system.
+
+        Throws: $(LREF ProcessException) if opening the process failed; this is
+        Windows specific.
+        */
```

Comment:
Actually, it seems like there's more to it - currently `wait` uses asserts to verify the validity of the Pid. If non-child PIDs becomes part of its API, I think those checks need to be made explicit.